src/routes/InventoryDetail.js

```diff
@@ -134,11 +134,13 @@ const Inventory = () => {
       entity &&
       appList[osSlug]?.map((app) => {
         app.isDisabled = app.nonEdge && hostType === 'edge' ? true : false;
+        setHType(hostType);
```

Comment:
I am wondering if there is any reason to set a state in a loop. If there is not, let's take this out.